main.js

```diff
@@ -17,4 +17,168 @@ mobileMenuItems.forEach((i) => {
+    tags: ['html', 'Ruby on rails', 'css', 'javascript'],
+    popupTags: ['html', 'css', 'javaScript', 'github', 'Ruby', 'Bootstrap'],
+    image: 'pics/homepage4.png',
+    desktopImage: 'pics/homepage4-desktop.png',
+    seeLive: '#',
+    seeSource: '#',
+  },
+];
+
+function subtitleValue(i) {
+  let subtitleList = [];
+  for (let j = 0; j < data[i].subtitle.length; j += 1) { subtitleList += `<li class="subtitle-${j}"> ${data[i].subtitle[j]} </li>`; }
+  return (subtitleList);
+}
+
+function tagsValue(i) {
+  let tagsList = [];
+  for (let j = 0; j < data[i].tags.length; j += 1) { tagsList += `<li class="tags-${j}"> ${data[i].tags[j]} </li>`; }
+  return tagsList;
+}
+
+function popupTagsValue(i) {
+  let tagsList = [];
+  for (let j = 0; j < data[i].tags.length; j += 1) { tagsList += `<li class="tool"> ${data[i].tags[j]} </li>`; }
+  return tagsList;
+}
+
+for (let i = 0; i < data.length; i += 1) {
+  document.querySelector('.work-container').innerHTML += `
+<div id="work-${i}" class="work-tonic">
+<div class="work-img-div">
+        <img src="${data[i].image}" alt="Project image" class="work-img"/>
+        <img src="${data[i].desktopImage}" alt="Project image" class="desk-work-img"/>
+      </div>
+      <div class="work-info">
+        <h2 class="tonic">${data[i].title}</h2>
+        <div class="text-info">
+          <ul>
+           ${subtitleValue(i)}
+          </ul>
+        </div>
+        <p class="work-text"> ${data[i].desription} </p>
+          <ul class="tags">
+            ${tagsValue(i)}
+              </ul>
+        
+        <button type="button" id="${data[i].id}" class='see-proj see-proj-${i}'>See project</button>
+      </div>
+    </div>`;
+}
+
+const arr = [];
+
+for (let i = 0; i < data.length; i += 1) {
+  arr.push(` <div class="modal">
+  <div class="popup-header">
+  <h2 class="tonic">${data[i].title}</h2>
+  <div id="close-popup-sign">
+      <span> <a  href="#" class="close-popUp">X</a></span>
+    </div>
+    </div>
+    <div class="subtitlePopupValue text-info">
+      <ul>
+        ${subtitleValue(i)}
+      <ul>
+    </div>
+  <div class="modal-div-3">
+    <div class="modal-img">
+    <img class="work-img1-mob" src="${data[i].image}" alt="work-img1" />
+    <img class="work-img1-desk" src="${data[i].desktopImage}" alt="work-img1" />
+    </div>
+    <div class="modal-info">
+    
+      <p class="modal-description desk mob">
+     ${data[i].popDescription}
+      </p>
+
+    <div class="tech">
+    <ul class="modal-tools desk">
+    ${popupTagsValue(i)}
+      </ul>
+      <ul class="modal-tools mobile">
+      ${popupTagsValue(i)}
+      </ul>
+      <button type="button" class="modal-btn">
+      See Live <i class="fa fa-external-link" aria-hidden="true"></i>
+      </button>
+      <button type="button" class="modal-btn">
+      See Source
+      <i class="fa fa-github-square" aria-hidden="true"></i>
+      </button>
```

Comment:
- [ ] Hi @davidobi, As already pointed out by the previous reviewer, let's make sure that these buttons on the desktop version should be aligne horizontally on a straight line. Once more check out the screenshots below:

Your Design | Figma Design
:---:|:---:
![](https://user-images.githubusercontent.com/63087491/156474136-02b95917-7bc3-4cb0-9d60-41194e1ffd38.jpeg) | ![](https://user-images.githubusercontent.com/63087491/156474159-b508d73e-db9b-49d9-8b5b-8043f40bd8fd.jpeg)

Reply:
@FrederickMih Thanks for your feedback. I am sorry I have maxed out all my attempts and as such, I cannot do anything on it. All the issues could have just been pointed out by the second reviewer. And now I cant do anything about it. Thanks for your feedback.

Reply:
I am sure if I fix this, another one would be pointed out. It's sometimes frustrating because I actually tried to fix it and it didn't work out. I have given up on it sorry

Reply:
- Hi @davidobi, you are really doing good in this project. You just need to put more effort again. Learning is all about receiving feedback. You don't need to give up because you are doing extremely well. You may reach out to me on slack if you still find any difficulties.

Reply:
Just correct the issue and resubmit. You haven't missed out all your attempts

Reply:
@FrederickMih Thanks for your feedback. I am sorry it is the end for me. I have never repeated a week before and If I would have to repeat because of this CSS issue then it's really sad. I rather withdraw from the program. Don't worry about it, I have written to student success. I don't even have the strength to write any extra code on this. And even if I do I cant submit. I would just get the sleep I deserve. Thank you and all the best in your Microverse Journey

Reply:
@davidobi, I want you to know something: 

Before finishing the Microverse curriculum, I repeated several weeks and even 1 complete module myself.  I had to understand that I hadn't really mastered that week/module curriculum, and those repetitions helped me a lot to really master the content.

If you are blocked or struggling with a part of the curriculum, there are ways you can get help, including us code reviewers.  If a review is not clear enough about the requirements, you are free to contact us here, or on Slack.

Maybe you have come across a concept called "The imposter syndrome".  Here is some [help](https://www.youtube.com/watch?v=RBcEpsTmT8Y).

Also, I'd like to tell you that everything in life that is worth (like a software development career) has a price, but that price is never too steep in comparison to what you will earn at the end of your journey.

> We shall not cease from exploration, and the end of all our exploring will be to arrive where we started and know the place for the first time. - T.S. Elliot

### You are not alone in your journey to be a professional software developer.
### Now please @davidobi, wake up, put your hands again on that keyboard, and never surrender again.